### DISCUSSION TITLE
개발자의 추상적, 구조적 사고 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#38] [Announcements]
# 개발자의 추상적, 구조적 사고 | kciter.so

컴퓨터 과학의 모든 문제는 다른 수준의 간접 계층으로 해결할 수 있다 - David J. Wheeler ... 그러나 그러면 또 다른 문제가 생기는 것이 일반적이다. - Butler W. Lampson 개발자는 추상과 구조에 대한 이야기를 많이…

https://kciter.so/posts/developers-abstraction-structural-thinking/

**Comment by kecan0406** [comment]
항상 좋은 글 작성해주셔서 감사합니다.

**Reply by kciter** [reply]
안녕하세요. 좋은 말씀 감사합니다. 글쓰기에 큰 원동력이 되네요 😄
앞으로도 도움이 될 수 있는 글 열심히 써보겠습니다!

**Comment by o5sy** [comment]
너무나 흥미로운 글들에 로그인을 하고 따봉을 눌렀습니다..
늘 생각에 대한 생각을 많이 하지만 정돈되지 않은 채 부유하는 것들이 많았는데 그런 내용을 단번에 정리해주네요.
인사이트 공유해주셔서 감사드립니다!
